packages/matchbox/src/components/Tabs/Tabs.js

```diff
@@ -77,26 +55,45 @@ class Tabs extends Component {
     }
   }
 
-  render() {
-    const { tabs, selected, connectBelow, color, fitted } = this.props;
-
-    const tabMarkup = tabs.map((tab, i) => (
-      <Tab key={i} index={i} fittedTab={fitted} selected={selected} {...tab} onClick={this.handleClick} />
-    ));
-
-    const tabsClasses = classnames(
-      styles.Tabs,
-      styles[`color-${color}`],
-      connectBelow && styles.connectBelow,
-      fitted && styles.fitted
-    );
-
-    return (
-      <div className={tabsClasses}>
-        {tabMarkup}
-      </div>
-    );
-  }
+  return (
+    <Box display="flex" borderBottom="400">
+      {tabs.map((tab, i) => (
+        <Tab key={i} index={i} fitted={fitted} selected={selected} {...tab} onClick={handleClick} />
+      ))}
+    </Box>
+  );
 }
 
+Tabs.displayName = 'Tabs';
+Tabs.propTypes = {
+  /**
+   * Tab Content
+   * Actions that build the tabs. Most button and unstyled link props will work in here.
+   * e.g. { content: 'Label', onClick: callback() }
+   */
+  tabs: PropTypes.arrayOf(
+    PropTypes.shape({
+      content: PropTypes.node.isRequired,
+    }),
+  ),
+
+  /**
+   * Tab Color
+   */
+  color: deprecate(
+    PropTypes.oneOf(['orange', 'blue', 'navy', 'purple', 'red']),
+    'Tab color is no longer configurable',
+  ),
+  /**
+   * Index of selected tab
+   */
+  selected: PropTypes.number.isRequired,
+  /**
+   * Connects this component with component underneath it. Works well with Panels.
+   */
+  connectBelow: deprecate(PropTypes.bool, 'Deprecated in favor of margin system props'),
```

Comment:
It looks like this style was removed.  If so, this should be removed or if you want to provide a nice DX, you could provide an error if it is being used.

Reply:
Thats what this `deprecate` function does 😄

Reply:
`deprecate` logs a warning.  That is what I would expect from a prop that is planned to be removed in a future release.  However, the supporting code for this props has been removed.  It would be inaccurate to log a warning.  Either this prop should be removed or it should throw an error to immediately alert the dev that it has been removed.

Reply:
Not 100% sure how to handle that – it's been replaced with system props. Would need to throw an error if the prop is _not_ used. But we're doing the same with almost every other component, so I think it can maybe be announced elsewhere.